arch/arm64/configs/sdm660_defconfig

```diff
@@ -1,5 +1,6 @@
 CONFIG_LOCALVERSION="-sdm660"
 # CONFIG_LOCALVERSION_AUTO is not set
+CONFIG_KERNEL_ZSTD=y
```

Comment:
Was ZSTD as kernel compression mode chosen automatically or manually, if manually then why?

Reply:
This selects HAVE_KERNEL_ZSTD so I think it was automatic https://www.kernelconfig.io/CONFIG_EFI_ZBOOT?q=Zboot&kernelversion=6.15.9&arch=x86

Reply:
It seems that Image.gz is still built using gzip, and zstd is used in efi-app self decompressor
```
  LD      vmlinux
  NM      System.map
  SORTTAB vmlinux
  OBJCOPY arch/arm64/boot/Image
  GZIP    arch/arm64/boot/Image.gz
  PAD     arch/arm64/boot/vmlinux.bin
  ZSTD22  arch/arm64/boot/vmlinuz
  OBJCOPY arch/arm64/boot/vmlinuz.o
  LD      arch/arm64/boot/vmlinuz.efi.elf
  OBJCOPY arch/arm64/boot/vmlinuz.efi
make[1]: Leaving directory '/home/runner/_work/linux/linux/.output'
```
just curious why simply enabling EFI causes so many changes to other Kconfigs.. :thinking: